src/panels/lovelace/hui-root.ts

```diff
@@ -743,12 +743,14 @@ class HUIRoot extends LitElement {
     const curViewConfig =
       typeof this._curView === "number" ? views[this._curView] : undefined;
 
-    if (curViewConfig?.back_path) {
-      navigate(curViewConfig.back_path);
+    if (curViewConfig?.back_path != null) {
+      navigate(curViewConfig.back_path, { replace: true });
```

Comment:
replace is actually wrong here... I guess the most correct would be to do a back and then replace... but... that would also not be correct...

Reply:
Yes, I added replace option to avoid pushing into history. But I'm ok to remove it.